docs/release-notes/cloud/cloud-2024-04.md

```diff
@@ -0,0 +1,100 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2024.
+included_jira_versions: ['2024.04']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.272
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.305
+---
+
+# Cloud April 2024
+
+These release notes are for the Codacy Cloud updates during April 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues with disabled release notes
+
+Bugs and other issues:
+-   https://codacy.atlassian.net/browse/ALA-1000
```

Comment:
@nicklem Please validate if we should include this one.

Reply:
[Following up](https://codacy.slack.com/archives/C04EJUDAL8J/p1714744232731459) with the squad

Reply:
Yes, let's:

```
You can now manually trigger a coverage pull request analysis using the [reanalyzeCoverage](https://api.codacy.com/api/api-docs#reanalyzecoverage) API endpoint. This can help you unblock pull requests that expect to receive a coverage report but only change uncovered files.
```

Reply:
Added! 👍